include/swift/Option/FrontendOptions.td

```diff
@@ -214,6 +214,11 @@ def disable_objc_attr_requires_foundation_module :
 def enable_resilience : Flag<["-"], "enable-resilience">,
    HelpText<"Deprecated, use -enable-library-evolution instead">;
 
+def disable_implicit_swift_modules: Flag<["-"], "disable-implicit-swift-modules">,
+  HelpText<"Disable building Swift modules explicitly by the compiler">;
+
+def swift_module_file_EQ : Joined<["-"], "swift-module-file=">,
```

Comment:
I think we've been trying to move toward "separate" flags nowadays, rather than joined flags with `=`.

Reply:
ah, will fix